### DISCUSSION TITLE
The actions bar at the bottom should be responsive to modifier keypresses

### DISCUSSION THREAD

**Discussion by goyalyashpal** [doublecmd/doublecmd#1087] [Suggestions]
### Description

A very small quality of life improvement: 

having this bar respond/update the action/keypress to the modifer keypresses (shift-/ctrl- etc) will improve discoverability & rememberance of those actions.

<img src="https://user-images.githubusercontent.com/19423063/222925583-e17b3d29-60cd-44cf-9cfb-ebafb88c3a39.png" width=500px/>

![gimp video link](https://user-images.githubusercontent.com/19423063/222925690-d25f4248-a48a-4988-a806-e10e20cad80f.mp4)

**Comment by goyalyashpal** [comment]
an issue on related matter, but tangential: https://github.com/doublecmd/doublecmd/issues/95

**Comment by goyalyashpal** [comment]
this was inspired from mucommander - but can't share screenshots/screenrecords for reference as i don't use it anymore

**Comment by alexx2000** [comment]
Really in DC it is not a function keys buttons, it is actions buttons. They show which hotkey assigned to actions: View, Edit, Copy etc.
